src/libcore/cell.rs

```diff
@@ -225,10 +225,10 @@ use ptr;
 ///
 /// let new_value = 100;
 ///
-/// // ERROR, because my_struct is immutable
+/// // ERROR: my_struct is immutable
 /// // my_struct.regular_field = new_value;
 ///
-/// // WORKS, although `my_struct` is immutable, field `special_field` is mutable because it is Cell
```

Comment:
Here, the old version is clearer because it explicitly says "is mutable because". (Cleaning up the grammar would be helpful, though.)